src/linter/ui5Types/asyncComponentFlags.ts

```diff
@@ -346,9 +354,63 @@ function reportUiComponentResults({
 	manifestContent: string | undefined;
 	resourcePath: string;
 }) {
-	const {hasAsyncInterface, routingAsyncFlag, rootViewAsyncFlag} = analysisResult;
+	const {hasAsyncInterface, routingAsyncFlag, rootViewAsyncFlag, manifestVersion} = analysisResult;
 	const componentFileName = path.basename(resourcePath);
 
+	// Determine manifest version from external manifest.json or inline manifest
+	const isManifestV2 = manifestVersion?.startsWith("2.") ?? false;
+
+	if (isManifestV2) {
+		// Only report errors when no external manifest.json exists to avoid duplication with ManifestLinter
+		if (manifestContent) {
+			// Avoid duplicate checks
+			return;
+		}
+		// Inline manifest definition (within Component.js)
+		// Manifest v2 logic:
+		// - Do not enforce IAsyncContentCreation (if present, do nothing)
```

Comment:
I don't see that the existence of the interface would have an impact on the below checks. To me, the comment seems outdated.